### PR TITLE
Raft trace log detail

### PR DESCRIPTION
**Pull request by manadart**
Adds some detail to the trace log output, in order to enhance Raft diagnostics.

## QA steps

- `juju bootstrap lxd test-lease --debug --config logging-config="<root>=DEBUG;juju.worker.lease.raft=TRACE;juju.core.raftlease=TRACE"`
- See the output with `juju debug-log -m controller --include-module juju.core.raftlease --include-module juju.worker.lease.raft`

## Documentation changes

None.

## Bug reference

Relevant to https://bugs.launchpad.net/juju/+bug/1900724, though not a fix.

### CODE REVIEW

File: cmd/jujud/agent/machine/manifolds.go [hunk 1]
```diff
@@ -379,19 +379,6 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 			NewWorker:              prworker.NewWorker,
 		}),
 
-		/* TODO(menn0) - this is currently unused, pending further
```
Comment (manadart) [inline cmd/jujud/agent/machine/manifolds.go:382]: Drive-by. This has been around for years, and unlikely to come back.

**Review by SimonRichardson — APPROVED** [review]

**Comment by manadart** [comment]
$$merge$$
